docs/pages/guides/testing-rsc.mdx

```diff
@@ -0,0 +1,119 @@
+---
+title: Testing React Server Components
+sidebar_title: Testing RSC
+description: Learn about writing unit tests for React and React Native Server Components in Expo.
+platforms: ['android', 'ios', 'web']
+---
+
+> This guide refers to the experimental feature React Server Components which is still in development.
+
+React Server Components (RSC) are a new feature in React that allows you to build components that render on the server and can be hydrated on the client.
+
+Expo has support for testing React Server Components in your project.
+
+## Jest Testing
+
+Unlike client-based Jest testing, React Server Components run in a Node.js environment, meaning you can more closely emulate the server-side rendering environment with Jest on its own.
+
+### Setup
+
+While standard server rendering is web-only, Expo's universal React Server Components bundle custom server renderers for each platform. This means platform-extensions like `*.ios.js` and `*.native.ts` will be resolved when writing server components for an iOS app.
```

Comment:
```suggestion
While standard server rendering is web-only, Expo's universal RSC bundles custom server renderers for each platform. For example, when writing server components for an iOS app, the platform-specific extensions such as **\*.ios.js** and **\*.native.ts** will be resolved.
```

- Fix grammatical error
- We can use the short form for React Server Components since that is already defined in the introduction for brevity.
- The second sentence targets only one native platform, so it portrays an example here. I've updated the above suggestion. However, if we want to portray both Android and iOS, we can rephrase it as:

```md
When writing server components for an Android or iOS app, the platform-specific extensions such as **\*.android.js**, **\*.ios.js**, and **\*.native.ts** will be resolved.
```